src/resources/components/SummaryDatapoint.vue

```diff
@@ -4,7 +4,13 @@
 	export default {
 		name: 'summary-datapoint',
 		props: {
-			changeValue: Number,
+			data: {
```

Comment:
@clowerweb check formatting. There's files that seem to be indented with tabs, and some that use spaces, and some use both.

Reply:
Those were fixed in other commits as requested.